.gitlab/kernel_version_testing/security_agent.yml

```diff
@@ -186,30 +162,27 @@ kernel_matrix_testing_run_security_agent_tests_arm64:
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/system-probe_arm64$DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_SYSPROBE_BUILDIMAGES
   tags: ["arch:arm64"]
   needs:
-    [
-      "kernel_matrix_testing_setup_env_security_agent_arm64",
-      "upload_dependencies_security_agent_arm64",
-    ]
+    - kernel_matrix_testing_setup_env_security_agent_arm64
+    - upload_dependencies_security_agent_arm64
+    - upload_security_agent_tests_arm64
   timeout: 3h
   variables:
     ARCH: "arm64"
   parallel:
     matrix:
       - TAG:
-          [
-            "ubuntu_18.04",
-            "ubuntu_20.04",
-            "ubuntu_22.04",
-            "ubuntu_23.10",
-            "amzn_5.4",
-            "amzn_5.10",
-            "fedora_37",
-            "fedora_38",
-            "debian_10",
-            "debian_11",
-            "debian_12",
-            "centos_79",
-          ]
+          - "ubuntu_18.04"
```

Comment:
nit: I notice quotes here, but not above. Are they needed?

Reply:
I think they are not 100% needed but just to be safe I prefer to quote strings in YAML that contain "special" characters like dot, that way I'm completely sure they're not getting interpreted as another type. Adding quotes to the ones above for consistency.